sources/packages/web/src/components/common/SearchStudentsCommon.vue

```diff
@@ -0,0 +1,208 @@
+<template>
+  <full-page-container>
+    <body-header
+      title="Search Students"
+      subTitle="Lookup students by entering their information below."
+    >
+    </body-header>
+    <v-form ref="searchStudentsForm">
```

Comment:
I am not sure if the shared component should be at the `full-page-container`. I would prefer to have it at the `v-form` level and the `full-page-container` and `body-header` would be defined at the specific client level.